pages/governance/proposal/[proposalId].governance.tsx

```diff
@@ -342,12 +342,14 @@ export default function ProposalPage({
                       <FormattedNumber
                         value={yaeVotes}
                         visibleDecimals={2}
+                        roundDown={true}
```

Comment:
All these `roundDown={true}` can just be simplified to `roundDown`

Reply:
Done